src/ai/backend/manager/server.py

```diff
@@ -281,6 +292,83 @@ async def manager_status_ctx(root_ctx: RootContext) -> AsyncIterator[None]:
     yield
 
 
+@actxmgr
+async def leader_election_ctx(root_ctx: RootContext) -> AsyncIterator[None]:
+    configuration = root_ctx.local_config["manager"]["raft-configuration"]
+    raft_id = configuration[root_ctx.pidx]
```

Comment:
Good point. It would be better to set its address manually in `manager.toml`.

Reply:
But we still have to distinguish each process on the same machine since they share the same `manager.toml`.
How about separating them into `local-raft-configuration` and `remote-raft-configuration`? Each process can concatenate them before passing them to a `Raft` instance.

Reply:
hmm, then each manager-runner machine has different `remote-raft-configuration` configuration.
It seems error prone cause we need to believe the user set a number of different configuration correctly.
I think that we need to have *fixed* machine id or ip addresses of machines.